pkg/visor/visorconfig/v1.go

```diff
@@ -105,8 +105,7 @@ type V1UptimeTracker struct {
 
 // V1ServiceDisc configures Skywire App Discovery Clients.
 type V1ServiceDisc struct {
-	UpdateInterval Duration `json:"update_interval,omitempty"`
-	ServiceDisc    string   `json:"service_discovery"`
+	ServiceDisc string `json:"service_discovery"`
```

Comment:
Since we only have a URL now we can get rid of the struct and just add `ServiceDisc` field to `V1Launcher` @jdknives

Reply:
I'm agree with you and I do these change.